modules/apps/29-fee/keeper/keeper_test.go

```diff
@@ -57,6 +63,14 @@ func SetupFeePath(path *ibctesting.Path) error {
 	return nil
 }
 
+func (suite *KeeperTestSuite) NewPacketId(id uint64) channeltypes.PacketId {
```

Comment:
This should actually go into main and update the channel types here probably -> https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/types/packet.go 

I think you can leave this function out for now and I'll make an issue for it and tag @colin-axner

Reply:
same comment as above but with channel-id

Reply:
Having it in the testing file is just duplicate code. We should have these in types similar to all of our other proto helpers. 

You can do this in a separate PR though. Agree with @damiannolan  :+1: